### PR TITLE
Update community-plugins.json

### PR DESCRIPTION
**Pull request by Marvy101**
Added a plugin!

lfg

### CODE REVIEW

File: community-plugins.json [hunk 1]
```diff
@@ -40,5 +40,13 @@
         "author": "Claude",
         "description": "Concise Summary of Patient's Medical History",
         "prompt": "You will receive a conversation between a healthcare provider and a patient discussing the patient's medical history. Your task is to carefully review the conversation and identify key information about the patient's past and current health conditions, surgeries, medications, allergies, family history, and any other relevant medical details mentioned. Use this information to generate a clear, concise, and well-organized summary of the patient's medical history. The summary should be divided into appropriate sections such as 'Past Medical History', 'Surgical History', 'Medications', 'Allergies', 'Family History', and 'Social History'. Ensure that the summary is written using professional medical terminology and maintains patient confidentiality. If any important aspects of the medical history are not discussed in the conversation, note them as 'Not Mentioned' under the relevant section headings."
-    }
+    },
+    {
+        "id": "ai-companion",
+        "name": "Her",
```
Comment (francip) [inline community-plugins.json]: This name is a bit of potential copyright issue...

Comment (Marvy101) [reply community-plugins.json]: fixed it

**Comment by akshaynarisetti** [comment]
Had to revert this for now, Please follow instructions [here](https://docs.basedhardware.com/developer/Plugins), happy to accept a pull request with revised the version @Marvy101

**Comment by Marvy101** [comment]
why was this reversed? what part of the instructions did I not follow?

Best,
Dada



[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
06/03/24,
03:11:37 AM

**Comment by akshaynarisetti** [comment]
Your prompt contains "asking questions about their day", which implies back and forth communication. Have a look at https://docs.basedhardware.com/developer/Plugins/#bad-prompt-example for more info.

**Comment by Marvy101** [comment]
updated it!

Best,
Dada



[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
06/03/24,
11:48:40 AM

**Comment by akshaynarisetti** [comment]
You should open a new pull request, thanks!

**Pull request merged**